src/readme.txt

```diff
@@ -54,15 +54,15 @@ Growing the BuddyPress community means better software for everyone!
 
 To run BuddyPress, we recommend your host supports:
 
-* PHP version 7.4 or greater.
+* PHP version 8.1 or greater.
```

Comment:
```suggestion
* PHP version 7.4 or greater.
```

Let's follow how WordPress evolves here too: [7.4 is still minimum recommended PHP version](https://wordpress.org/about/requirements/). Let's keep it this way.

Reply:
This is actually the recommended PHP version, not the minumum.

Reply:
Ops. I see. That's correct!